rush-state/src/console.rs

```diff
@@ -80,6 +80,15 @@ pub struct Console<'a> {
     data: ConsoleData<'a>,
 }
 
+#[derive(Debug)]
```

Comment:
This is interesting but I would stash these changes for a separate PR as it is not really related to error handling from what I can tell.